tictactoe/src/main/java/models/TicTacToeService.java

```diff
@@ -0,0 +1,144 @@
+package main.java.models;
+
+import java.util.List;
+
+public class TicTacToeService {
+  private final GameBoard gameboard = new GameBoard();
+
+  public void setNoOfColumnsGameBoard(int setNoOfColumnsGameBoard) {
+    gameboard.setNoOfColumnsGameBoard(setNoOfColumnsGameBoard);
+  }
+
+  public void setNoOfRowsGameBoard(int setNoOfRowsGameBoard) {
+    gameboard.setNoOfRowsGameBoard(setNoOfRowsGameBoard);
+  }
+
+  public void setPlayers(List<Player> players) {
+    gameboard.setPlayers(players);
+  }
+
+  public void setUserRowColumnInputs(List<UserRowColumnInput> userRowColumnInputs) {
+    gameboard.setUserRowColumnInputs(userRowColumnInputs);
+  }
+
+  public void setGrid(int noOfRowsGameBoard, int noOfColumnsGameBoard) {
+    gameboard.setGrid(noOfRowsGameBoard,noOfColumnsGameBoard);
+  }
+
+  public void startGame() {
+
+    String [][]grid = gameboard.getGrid();
+    initialiseGameBoard(grid);
```

Comment:
Why pass grid as parameter?
These functions would already have access to gameboard

Reply:
yes, got it